### ISSUE TITLE
Icons do not load on my github

### ISSUE THREAD

**Issue by lporras** [tgirardi/github-open-in-sublime#2]
Hey, Awesome extension.  I have followed all the steps to install it on my pc, but when I go to a file on my repo the buttons that adds the extension do not load the icon images.

![screen shot 2015-02-23 at 10 42 32 am](https://cloud.githubusercontent.com/assets/232293/6328703/d6af7baa-bb48-11e4-884e-a7a118d90550.png)

**Comment by tgirardi** [comment]
Hi @lporras, the icons are provided by [Font Awesome](http://fontawesome.io), I'll check if I had made an error in the release when including that dependency.

In the mean time, could you check in the chrome web dev tools' console if there's an error printed there; and in the network section if there's a 404? Thanks!

**Comment by tgirardi** [comment]
@lporras: Could you apply the following patch to the content script's folder (the one being used by chrome to load the extension), reload the extension and see if the problem is solved?

``` patch
diff --git a/github-sublime.js b/github-sublime.js
index 62bb8ed..c2a845e 100644
--- a/github-sublime.js
+++ b/github-sublime.js
@@ -1,6 +1,14 @@
 (function ($, chrome) {
   'use strict';

+  // add Font Awesome @font-face (workaround)
+  var styleNode = document.createElement ('style');
+  styleNode.type = 'text/css';
+  styleNode.textContent = '@font-face { font-family: FontAwesome; src: url("' +
+    chrome.extension.getURL ('lib/fontawesome/fontawesome-webfont.woff') +
+    '"); }';
+  document.head.appendChild (styleNode);
+
   // jQuery selector for "desired" lines
   var LINE_SELECTOR =
     'tr.file-diff-line,div.line,.diff-table tr,.blob-wrapper tr';
```

I don't know why my browser does show the icons, so I need someone experiencing the problem to test the patch.

**Comment by tgirardi** [comment]
Solved in new release 0.0.3: https://github.com/tgirardi/github-open-in-sublime/releases/tag/0.0.3
